presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/compose/BottomNavigation.kt

```diff
@@ -13,12 +13,17 @@ fun BottomNavigation(
     onNextClicked: () -> Unit,
     nextText: String? = null,
     backEnabled: Boolean = true,
+    backVisible: Boolean = true,
     nextEnabled: Boolean = true
 ) {
-    Row(modifier = Modifier
-        .padding(top = 10.dp, bottom = 10.dp)
-        .fillMaxWidth()) {
-        WhiteBackButton(onClick = onBackClicked, enabled = backEnabled)
+    Row(
+        modifier = Modifier
+            .padding(vertical = 10.dp)
+            .padding(horizontal = 20.dp)
```

Comment:
Since the BottomNavigation is always used in a Column layout, does the horizontal padding do anything? Or can we remove that line?

Reply:
Adding padding to the Column layout that BottomNavigation is in will cause the image to also be padded. It seems that unless I isolate the BottomNavigation the horizontal padding has to be added to the BottomNavigation itself instead of the Column one level up from BottomNavigation.
<img width="264" alt="Screen Shot 2022-10-28 at 11 19 39 AM" src="https://user-images.githubusercontent.com/84655410/198705474-9c827fcf-af77-4eb5-9181-2056ab52e1b5.png">